app/windows/runner/win32_window.h

```
@@ -0,0 +1,102 @@
+#define RUNNER_WIN32_WINDOW_H_
+
+#include <windows.h>
+
+#include <functional>
+#include <memory>
+#include <string>
+
+// A class abstraction for a high DPI-aware Win32 Window. Intended to be
+// inherited from by classes that wish to specialize with custom
+// rendering and input handling
+class Win32Window {
```

Clang (14.0.6)

[warning] variable 'Win32Window' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)